directories/src/redux/epics/treasure-hunt-epics.js

```diff
@@ -9,33 +9,43 @@ import Datamap from "../../utils/datamap";
 import Sidechain from "../../utils/sidechain";
 import Encryption from "../../utils/encryption";
 
+import { CHUNKS_PER_SECTOR } from "../../config/";
+
 const performPowEpic = (action$, store) => {
   return action$
     .ofType(treasureHuntActions.TREASURE_HUNT_PERFORM_POW)
     .mergeMap(action => {
+      console.log("pppppppppppppppp");
       const { treasureHunt } = store.getState();
-      const {
-        address,
-        message,
-        treasure,
-        chunkIdx,
-        numberOfChunks
-      } = treasureHunt;
+      const { dataMapHash, treasure, chunkIdx, numberOfChunks } = treasureHunt;
+
+      // const address = Encryption.obfuscate(dataMapHash);
```

Comment:
not sure if this works yet, need to QA it later when we have real genesis hashes / number of chunks